db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -306,13 +309,22 @@ private void parsePRDDTA() throws DRDAProtocolException
         initialize();
     }
 
+    private String getSessionName() {
+        return session == null ? "" : session.toString();
+    }
+
+    private void debug(String message) {
+        if(log.isDebugEnabled()) {
```

Comment:
The reason we redundantly check if Debug/Trace levels are enabled outside the logging methods is to avoid argument evaluation which is done unconditionally and may have quite noticeable overhead.
Introducing an intermediate convenience method apparently defeats the purpose. 
The existing _trace()_ method doesn't conform to that principle and should be reworked (separately from this PR; also to use Logger). It should not be considered as acceptable still.

Reply:
Yes, I also think we should change `trace()` to use our `Logger` but I would do it separately since it is orthogonal to this change.